runtime/src/test/java/com/redhat/ipaas/runtime/ComponentsITCase.java

```diff
@@ -26,15 +26,15 @@
     @Test
     public void componentsListTest() {
         ListResult<Component> results = restTemplate().getForObject("/api/v1/components", ListResult.class);
-        assertThat(results.getItems()).hasSize(20);
-        assertThat(results.getTotalCount()).isEqualTo(50);
+        assertThat(results.getItems()).hasSize(2);
+        assertThat(results.getTotalCount()).isEqualTo(2);
     }
 
     @Test
     public void componentsGetTest() {
-        Component result = restTemplate().getForObject("/api/v1/components/1", Component.class);
+        Component result = restTemplate().getForObject("/api/v1/components/org.foo:twitter-mention-connector:1.0", Component.class);
```

Comment:
1. I'm all for keeping things consistent, though I had not heard we renamed Component to Connector, last I heard we were naming the 'Apache Connector' a 'Component' in the UI. 
2. When will be adding in 'actions'?
3. BTW, I don't think 'Components' are a part of the demo directly as we are focussing on Connections and Integrations for the demo? So maybe we should wait with renaming till after the demo?